src/accessibility.md

```diff
@@ -122,7 +122,9 @@ There are currently no known accessibility concerns in the GOV.UK Frontend Docum
 
 From the GOV.UK Frontend codebase:
 
-The accordion component is currently using an `aria-labelledby` ARIA attribute on an unsupported `<div>` element. This is an incorrect implementation of [WAI-ARIA 1.1](https://www.w3.org/TR/wai-aria-1.1/), but is not a failure of WCAG success criterion [4.1.2 Name, Role, Value](https://www.w3.org/TR/WCAG21/#name-role-value). We plan to fix this within 2023. Track our progress on the [GitHub issue: ‘Accordion - Elements must only use allowed ARIA attributes’](https://github.com/alphagov/govuk-frontend/issues/2472).
+1. The accordion component is currently using an `aria-labelledby` ARIA attribute on an unsupported `<div>` element. This is an incorrect implementation of [WAI-ARIA 1.1](https://www.w3.org/TR/wai-aria-1.1/), but is not a failure of WCAG success criterion [4.1.2 Name, Role, Value](https://www.w3.org/TR/WCAG21/#name-role-value). We plan to fix this within 2023. Track our progress on the [GitHub issue: ‘Accordion - Elements must only use allowed ARIA attributes’](https://github.com/alphagov/govuk-frontend/issues/2472).
+2. The details component doesn't work well with Dragon and VoiceOver. This is a known issue with the assistive tech and the <details> HTML element, rather than an issue with our details component specifically. Track our progress on the [GitHub issue: 'Details component doesn't work well with Dragon and VoiceOver'](https://github.com/alphagov/govuk-frontend/issues/3693).
```

Comment:
```suggestion
2. The details component doesn't work well with Dragon and VoiceOver. This is a known issue with the assistive tech and the `<details>` HTML element, rather than an issue with our details component specifically. Track our progress on the [GitHub issue: 'Details component doesn't work well with Dragon and VoiceOver'](https://github.com/alphagov/govuk-frontend/issues/3693).
```

Code comment `<details>` so the build process doesn't attempt to parse it as HTML.

Reply:
Thank you for spotting that :)